commit b710648dee63466be01b34b21d929ada422f2845
Author: Bill Nottingham <notting@redhat.com>
Date:   Mon Jul 25 13:34:12 2011 -0400

    Remove fedora-sysinit-hack/fedora-sysinit-unhack; they're not needed with /var/lock on tmpfs.

diff --git a/Makefile b/Makefile
index 718b451..1e8c09e 100644
--- a/Makefile
+++ b/Makefile
@@ -127,11 +127,9 @@ install:
 	ln -s ../fedora-autoswap.service $(ROOT)/lib/systemd/system/basic.target.wants
 	ln -s ../fedora-autorelabel.service $(ROOT)/lib/systemd/system/basic.target.wants
 	ln -s ../fedora-autorelabel-mark.service $(ROOT)/lib/systemd/system/basic.target.wants
-	ln -s ../fedora-sysinit-hack.service $(ROOT)/lib/systemd/system/basic.target.wants
 	ln -s ../fedora-readonly.service $(ROOT)/lib/systemd/system/local-fs.target.wants
 	ln -s ../fedora-storage-init.service $(ROOT)/lib/systemd/system/local-fs.target.wants
 	ln -s ../fedora-storage-init-late.service $(ROOT)/lib/systemd/system/local-fs.target.wants
-	ln -s ../fedora-sysinit-unhack.service $(ROOT)/lib/systemd/system/multi-user.target.wants
 
 	mkdir -p $(ROOT)/etc/tmpfiles.d
 	install -m 644 initscripts.tmpfiles.d $(ROOT)/etc/tmpfiles.d/initscripts.conf
diff --git a/systemd/system/fedora-sysinit-hack.service b/systemd/system/fedora-sysinit-hack.service
deleted file mode 100644
index dd78da8..0000000
--- a/systemd/system/fedora-sysinit-hack.service
+++ /dev/null
@@ -1,10 +0,0 @@
-[Unit]
-Description=Bootup hack
-DefaultDependencies=no
-Conflicts=shutdown.target
-Before=basic.target shutdown.target
-
-[Service]
-Type=oneshot
-RemainAfterExit=yes
-ExecStart=/bin/touch /dev/.in_sysinit
diff --git a/systemd/system/fedora-sysinit-unhack.service b/systemd/system/fedora-sysinit-unhack.service
deleted file mode 100644
index 221d8bb..0000000
--- a/systemd/system/fedora-sysinit-unhack.service
+++ /dev/null
@@ -1,11 +0,0 @@
-[Unit]
-Description=Bootup unhack
-DefaultDependencies=no
-Conflicts=shutdown.target
-Before=shutdown.target
-After=fedora-sysinit-hack.service sysinit.target
-
-[Service]
-Type=oneshot
-RemainAfterExit=yes
-ExecStart=-/bin/rm -f /dev/.in_sysinit
